index.html

```diff
@@ -380,13 +380,6 @@ <h1>Introduction</h1>
       <li>a set of common deployment patterns in
         <a href="#sec-common-deployment-patterns"></a>,
       </li>
-      <li>
```

Comment:
Why did you remove that paragraph?

Reply:
Because "Introduction" contains a description of sections, not subsections.   
"Cross domain collaboration" and "System Integration" section are moved to subsections in Section 5
(5.10 and 5.11, respectively).